apisix/discovery/consul_kv.lua

```diff
@@ -331,6 +332,15 @@ function _M.connect(premature, consul_server)
             ", got result: ", json_delay_encode(result, true),
             ", with error: ", error_info)
 
+        if not retry_delay then
+            retry_delay = 1
+        else
+            retry_delay = retry_delay * 4
```

Comment:
Why here we use `4` as the factor?

Reply:
Discussed in https://github.com/apache/apisix/issues/4632#issuecomment-883184083

Reply:
2 is not large enough for me.